src/index.js

```diff
@@ -21,87 +21,96 @@ function encodeAttr(str) {
 }
 
 /** Parse Markdown into an HTML String. */
-export default function parse(md, prevLinks) {
-	let tokenizer = /((?:^|\n+)(?:\n---+|\* \*(?: \*)+)\n)|(?:^```(\w*)\n([\s\S]*?)\n```$)|((?:(?:^|\n+)(?:\t|  {2,}).+)+\n*)|((?:(?:^|\n)([>*+-]|\d+\.)\s+.*)+)|(?:\!\[([^\]]*?)\]\(([^\)]+?)\))|(\[)|(\](?:\(([^\)]+?)\))?)|(?:(?:^|\n+)([^\s].*)\n(\-{3,}|={3,})(?:\n+|$))|(?:(?:^|\n+)(#{1,3})\s*(.+)(?:\n+|$))|(?:`([^`].*?)`)|(  \n\n*|\n{2,}|__|\*\*|[_*])/gm,
-		context = [],
-		out = '',
-		links = prevLinks || {},
-		last = 0,
-		chunk, prev, token, inner, t;
+export default function snarkdown(md, options = {}) {
+	const highlight = options.highlight;
```

Comment:
Idea: since parsing is synchronous, rather than having to wrap `parse()` in a function to collect `options`, we could set a global `highlight` value that gets used by the other functions.  Right now this PR adds 100b, I'm thinking with that change we could trim it down to keep things below 1kb.  Thoughts?